frontend/packages/integration-tests-cypress/views/details-page.ts

```diff
@@ -18,4 +18,7 @@ export const detailsPage = {
 export namespace DetailsPageSelector {
   export const name = 'dd[data-test-selector="details-item-value__Name"]';
   export const namespace = 'dd[data-test-selector="details-item-value__Namespace"] a';
+  export const sectionHeadings = '[data-test-section-heading]';
+  export const itemLabels = 'dt';
+  export const horizNavTabs = '.co-m-horizontal-nav__menu-item';
```

Comment:
nit: I have a slight preference for spelling out "horizontal," particularly because that's how it's done elsewhere in the code.

```suggestion
  export const horizontalNavTabs = '.co-m-horizontal-nav__menu-item';
```

Reply:
fixed